Automattic/wp-calypso
# Domain: Move .page and .gay badges back to info icons and style them inline

Related to: https://github.com/Automattic/dotcom-forge/issues/4482

- This PR reverses the changes to .page & .gay info icons in Automattic/wp-calypso#83606 where they were turned into badges to make the info icons more subtle.
- It also adjusts the info icons so they are inline with domain name.

Before | After
--|--
<img width="1541" alt="Screenshot 2023-11-06 at 3 14 58 PM" src="https://github.com/Automattic/wp-calypso/assets/140841/4f2b1ecc-64c2-4926-bad4-1636d587900c"> | <img width="1540" alt="Screenshot 2023-11-06 at 3 15 58 PM" src="https://github.com/Automattic/wp-calypso/assets/140841/b6f285c9-c546-478e-9e29-526af24b1043">


## Testing Instructions



* Go to /domains and/or /domains/add/[site]
* Search for exampledomain.gay and set the filter to `.page`
* View the info icons and check that they look good and are correct
* Check in mobile
* Make sure the premium domain badge still looks good. You can search for insurance.blog to see one.
